glasswall/content_management/config_elements/sysConfig.py

```diff
@@ -8,16 +8,20 @@ class sysConfig(ConfigElement):
     """ A sysConfig ConfigElement.
 
     Key word arguments can be specified to change individual switch values:
-    sysConfig(interchange_type="xml", interchange_pretty="false")
+    sysConfig(interchange_type="sisl",interchange_pretty="true",)
     """
 
     def __init__(self, attributes={}, **kwargs):
         self.name = self.__class__.__name__
         self.attributes = attributes
+
         self.switches_module = switches.sys
         self.default_switches = [
             self.switches_module.interchange_type(value="sisl"),
             self.switches_module.interchange_pretty(value="false"),
```

Comment:
```suggestion
            self.switches_module.interchange_pretty(value="true"),
```

Reply:
It looked like this was the intended change based on the change to line 11. If this is not the case and the default value for the `interchange_pretty` switch is still false then this suggested change can be ignored.

Reply:
It (line 11) was a comment as an example. Engine default value is `false`